.github/workflows/yaml-checks.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: yamllint
+on: [push, pull_request]
+
+jobs:
+  yamllint:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/devel/openqa/containers/os-autoinst_dev
+    steps:
+      - uses: actions/checkout@v4
+      - name: Validate yamls
+        run: |
+          git config --global --add safe.directory '*'
+          yamllint --strict $(git ls-files "*.yml" "*.yaml" 2> /dev/null || find . -name '*.y*ml')
```

Comment:
Please use yamllint as already mentioned in the other PR:
https://github.com/os-autoinst/os-autoinst-common/blob/master/.github/workflows/yamllint.yml
You don't need to pass it the list of files, it will by default lint .yml and .yaml